client-go/cmd/apps.go

```diff
@@ -0,0 +1,245 @@
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/deis/deis/client-go/controller/client"
+	"github.com/deis/deis/client-go/controller/models/apps"
+	"github.com/deis/deis/client-go/controller/models/config"
+)
+
+// AppCreate creates an app with the application
+func AppCreate(id string, buildpack string, remote string, noRemote bool) error {
+	c, err := client.Load()
+
+	fmt.Print("Creating Application... ")
+	quit := progress()
+	app, err := apps.Create(*c, id)
+
+	quit <- true
+	<-quit
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("done, created %s\n", app.ID)
+
+	if buildpack != "" {
+		configValues := map[string]string{
+			"BUILDPACK_URL": buildpack,
+		}
+		err = config.Set(*c, app.ID, configValues)
+
+		if err != nil {
+			return err
+		}
+	}
+
+	if !noRemote {
+		return c.CreateRemote(remote, app.ID)
+	}
+
+	fmt.Println("remote available at", c.RemoteURL(app.ID))
+
+	return nil
+}
+
+// AppsList lists apps on the controller
+func AppsList() error {
+	c, err := client.Load()
+
+	if err != nil {
+		return err
+	}
+
+	apps, err := apps.List(*c)
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("=== Apps")
+
+	for _, app := range apps {
+		fmt.Println(app.ID)
+	}
+	return nil
+}
+
+// AppInfo prints info about app
+func AppInfo(appID string) error {
+	c, err := client.Load()
+
+	if err != nil {
+		return err
+	}
+
+	if appID == "" {
+		appID, err = c.DetectApp()
+
+		if err != nil {
+			return err
+		}
+	}
+
+	app, err := apps.Get(*c, appID)
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("=== %s Application\n", app.ID)
+	fmt.Println("updated: ", app.Updated)
```

Comment:
The old client would return the raw JSON object string, though I like this format better. However, if we're deviating from Heroku's look-and-feel here, I think it'd look better if this was tab-ified, much like Heroku's client:

```
$ heroku info -a polar-shelf-1435
=== polar-shelf-1435
Dynos:         0
Git URL:       https://git.heroku.com/polar-shelf-1435.git
Owner Email:   me@bacongobbler.com
Region:        us
Stack:         cedar-14
Web URL:       https://polar-shelf-1435.herokuapp.com/
Workers:       0
```